### PR TITLE
Integrating eula prompt with install/apply/load commands

### PR DESCRIPTION
**Pull request by andriy-bulynko**

### CODE REVIEW

**Review by ashwathishiva — APPROVED** [review]

File: cmd/qliksense/apply.go [hunk 1]
```diff
@@ -1,39 +1,39 @@
...
 	"github.com/qlik-oss/sense-installer/pkg/qliksense"
 	"github.com/spf13/cobra"
 )
 
 func applyCmd(q *qliksense.Qliksense) *cobra.Command {
-
+	opts := &qliksense.InstallCommandOptions{}
 	filePath := ""
+	keepPatchFiles := false
+	overwriteExistingContext := false
```
Comment (ffoysal) [inline cmd/qliksense/apply.go]: it should be true by default.

File: cmd/qliksense/apply.go [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
 	c := &cobra.Command{
 		Use:     "apply",
 		Short:   "install qliksense based on provided cr file",
 		Long:    `install qliksense based on provided cr file`,
-		Example: `qliksense apply -f file_name or cat cr_file | qliksense apply -f -`,
```
Comment (ffoysal) [inline cmd/qliksense/apply.go:18]: since you added pipe we can keep this example

File: cmd/qliksense/apply.go (hunk 1 above)
Comment (andriy-bulynko) [reply cmd/qliksense/apply.go]: addressed

File: cmd/qliksense/apply.go (hunk 2 above)
Comment (andriy-bulynko) [reply cmd/qliksense/apply.go:18]: addressed

**Review by ffoysal — APPROVED** [review]

**Pull request merged**